CONTRIBUTING.md

```diff
@@ -2,29 +2,29 @@
 
 Thank you for your contributing to SmartBear MCP!
 
-1. [Contributing](#contributing)
-   1. [Getting Started](#getting-started)
-      1. [Repository Structure](#repository-structure)
-   2. [Development Setup](#development-setup)
-   3. [Testing Guidelines](#testing-guidelines)
-      1. [Test Requirements](#test-requirements)
-      2. [Running Tests](#running-tests)
-   4. [Documentation](#documentation)
-   5. [Releases](#releases)
-   6. [Additional Resources](#additional-resources)
+- [Contributing](#contributing)
+  - [Getting Started](#getting-started)
+    - [Repository Structure](#repository-structure)
+  - [Development Setup](#development-setup)
+  - [Testing Guidelines](#testing-guidelines)
+    - [Test Requirements](#test-requirements)
+    - [Running Tests](#running-tests)
+  - [Documentation](#documentation)
+  - [Releases](#releases)
+  - [Additional Resources](#additional-resources)
```

Comment:
seems like vscode keeps changing this when i format the document, happy to revert it if we dont want this